src/__tests__/to-be-disabled.tsx

```diff
@@ -79,6 +91,20 @@ describe('.toBeEnabled', () => {
       expect(() => expect(queryByTestId(name)).not.toBeEnabled()).toThrow();
     });
   });
+
+  test('handle editable prop for TextInput when true', () => {
+    const { queryByTestId } = render(
+      <View testID="view">
```

Comment:
This testID is not needed
```suggestion
      <View>
```

Reply:
Should I get rid of all other useless testID in this file as well ?

Reply:
Yes pls

Reply:
done